src/tribler-core/tribler_core/components/tag/db/tag_db.py

```diff
@@ -124,11 +124,11 @@ def add_tag_operation(self, operation: TagOperation, signature: bytes, is_local_
             self.instance.TorrentTagOp(torrent_tag=torrent_tag, peer=peer, operation=operation.operation,
                                        clock=operation.clock, signature=signature)
             torrent_tag.update_counter(operation.operation, is_local_peer=is_local_peer)
-            return
+            return True
```

Comment:
**issue:** "Returns:" section presents in the docstring, but return value is not described

Reply:
Fixed